src/App.jsx

```diff
@@ -58,7 +58,7 @@ export function App() {
 						path="/list"
 						element={<List data={data} listToken={listToken} />}
 					/>
-					<Route path="/add-item" element={<AddItem listId={listToken} />} />
```

Comment:
Is there a reason why listToken is not being passed as a prop to `AddItem` and `Layout` ?